src/components/postPagination.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+    <div class="btn-con">
+        {{this.buttonsRen()}}
+    <button class="btn" v-for="button in buttons" @click="paginationAlgo(button)" :class="currentPage(button)" :key="button">{{ button }}</button>
+        </div>
+</template>
+
+<script>
+/*eslint-disable*/
+export default {
+    props:["posts","paginationAlgo", "activePage"],
```

Comment:
@ChoudharyHamza make meaningful variable names. `posts` prop does not represent the posts but count.

Reply:
ok will be carefull from now on

Reply:
@ChoudharyHamza once you fix the name I will approve the merge and you will be good to merge.

Reply:
ok made the changes to the project